tools/scripts/update_assets_metadata.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+import json
+import hashlib
+import re
+
+ASSETS_JSON = './rotkehlchen/data/all_assets.json'
+ASSETS_META = './rotkehlchen/data/all_assets.meta'
+ASSETS_TEST = './rotkehlchen/tests/unit/test_assets.py'
+
+with open(ASSETS_META, "r") as assets_meta:
+    old_meta = json.load(assets_meta)
+
+with open(ASSETS_JSON, "rb") as assets_json:
```

Comment:
Really need to use black to enforce common style without having to do it in reviews. We use `' '` and not `" "` in the code for string literals.